hedera-node/test-clients/src/eet/java/EndToEndTests.java

```diff
@@ -65,9 +69,9 @@ Collection<DynamicContainer> file() {
                 //				extractSpecsFromSuite(DiverseStateValidation::new),
                 extractSpecsFromSuite(ExchangeRateControlSuite::new),
                 //              extractSpecsFromSuite(FetchSystemFiles::new),
-                //              extractSpecsFromSuite(FileAppendSuite::new),
-                //              extractSpecsFromSuite(FileCreateSuite::new),
-                //				extractSpecsFromSuite(FileDeleteSuite::new),
+                extractSpecsFromSuite(FileAppendSuite::new),
+                extractSpecsFromSuite(FileCreateSuite::new),
+                extractSpecsFromSuite(FileDeleteSuite::new),
                 extractSpecsFromSuite(FileUpdateSuite::new),
```

Comment:
Good to see some of these re-enabled.
Always been a little odd that so many E2E tests are commented out.
Perhaps @rbair23 or @tinker-michaelj has some insight to why so few of these tests are enabled.

Reply:
Do these tests also succeed in all other environments where these tests are used?

Reply:
I removed it, and we will address it in separate PR after a discussion with @tinker-michaelj